openapi.yaml

```diff
@@ -8371,7 +8367,7 @@ paths:
                           items:
                             type: object
                             allOf:
-                            $ref: '#/components/schemas/NodeBalancerNode'
+                            - $ref: '#/components/schemas/NodeBalancerNode'
```

Comment:
allOf should accept an array, so I think we needed a dash here

Reply:
ok